server/keyspace/keyspace.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keyspace
+
+import (
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/keyspacepb"
+	"github.com/tikv/pd/pkg/syncutil"
+	"github.com/tikv/pd/server/id"
+	"github.com/tikv/pd/server/storage/endpoint"
+)
+
+const (
+	// AllocStep set idAllocator's step when write persistent window boundary.
+	// Use a lower value for denser idAllocation in the event of frequent pd leader change.
+	AllocStep = uint64(100)
+	// AllocLabel is used to label keyspace idAllocator's metrics.
+	AllocLabel = "keyspace-idAlloc"
+	// DefaultKeyspaceName is the name reserved for default keyspace.
+	DefaultKeyspaceName = "DEFAULT"
```

Comment:
Is there any reason to use big letters?

Reply:
nothing in particular, it's just feels kinda natural to use for name with special meaning.
Is there any style reference for this?

Reply:
No, AFAICT.